src/module-elasticsuite-analytics/view/adminhtml/layout/smile_elasticsuite_analytics_search_usage.xml

```diff
@@ -67,6 +58,15 @@
                             <argument name="help" xsi:type="string" translate="true"><![CDATA[The terms listed below were those for which they were neither exact results nor close matches.<br />If you have nonetheless some related products, you probably should add a rule in the <a href="https://github.com/Smile-SA/elasticsuite/wiki/ThesaurusAdministration" target="blank">Thesaurus</a>.<br />For instance, if you have a lot of "running" related products and visitors are searching repeatly for "jogging" without any results, you should definitely add a <a href="https://github.com/Smile-SA/elasticsuite/wiki/ThesaurusAdministration#synonym" target="_blank">synonym</a> or <a href="https://github.com/Smile-SA/elasticsuite/wiki/ThesaurusAdministration#expansion" target="_blank">expansion</a> rule in the Thesaurus to associate the terms "running" and "jogging".]]></argument>
                         </arguments>
                     </block>
+
+                    <block name="elasticsuite.analytics.search_usage.spellchecked_terms" class="Smile\ElasticsuiteAnalytics\Block\Adminhtml\Search\Usage\SearchTerms" template="Smile_ElasticsuiteAnalytics::search/usage/terms.phtml">
+                        <arguments>
+                            <argument name="report" xsi:type="object">Smile\ElasticsuiteAnalytics\Model\Search\Usage\Terms\SpellcheckedTerms\Report</argument>
+                            <argument name="show_conversion_rate" xsi:type="boolean">true</argument>
+                            <argument name="title" xsi:type="string" translate="true">Spellchecked search terms</argument>
+                            <argument name="help" xsi:type="string" translate="true"><![CDATA[The terms listed below were those for which they were no exact results and only close matches were displayed.<br />If a term is a "popular mispelling" with not many results while you know the correct search term would return more products, you might want to add a rule in the <a href="https://github.com/Smile-SA/elasticsuite/wiki/ThesaurusAdministration" target="blank">Thesaurus</a>.]]></argument>
+                        </arguments>
+                    </block>
```

Comment:
Please use the "after" attribute of the block tag to avoid changing the block definition location